src/rules/no-empty-file/README.md

```diff
@@ -0,0 +1,33 @@
+# no-empty-file
+
+Disallow empty files.
```

Comment:
Are we also disallowing files that are only whitespace (not *completely* empty)? If so we should make that clear.